autobahn/wamp/uri.py

```diff
@@ -282,29 +283,35 @@ def is_exception(self):
 
 
 @public
-def register(uri):
+def register(uri, **options):
     """
```

Comment:
I feel like this would leave us with decorators that look a little bloated but I guess it would also make the usage more consistent with that of the `session.register` and `session.subscribe` methods. Shall I make this change then?